docs/language/contracts.rst

```diff
@@ -31,10 +31,22 @@ Instantiation using new
 _______________________
 
 Contracts can be created using the ``new`` keyword. The contract that is being created might have
-constructor arguments, which need to be provided.
+constructor arguments, which need to be provided. On Solana, contracts cannot exist as a variable, so it is not
+possible to instantiate a contract with the new keyword. Instead, a function ``new`` is made available to call its
```

Comment:
"On Solana, contracts cannot exist as a variable, so it is not possible to instantiate a contract with the new keyword. " That does not follow.